google-cloud-testing/storage-benchwrapper/pom.xml

```diff
@@ -0,0 +1,124 @@
+<?xml version="1.0"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <artifactId>benchwrapper</artifactId>
+  <version>0.0.0</version>
```

Comment:
At least 0.1.0 and perhaps 1.0.0

Reply:
0.0.0 is intentional: this is never going to be released, so the version should look like something fake. If version weren't a required field, I would delete it.